### ISSUE TITLE
Better indicate links in text: 1

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#264]
Either make contrast ratio between plain and linked text greater than 3:1 or underline, italicize, or bold the links. Current hover/focus behavior is fine, but there needs to be some kind of passive indication that works for color blind users.

> Links within the main content should be easily identified. The contrast ratio between the link text and surrounding non-link text should be greater than 3:1 in addition to hover and focus indication in the form of font style changes or the addition of an underline. An alternative would be to underline, italicize, or bold the links.

![screen shot 2018-04-27 at 2 36 17 pm](https://user-images.githubusercontent.com/263285/39381614-5f8df688-4a28-11e8-82b8-2090a3f62780.png)

> Above, you can see how the links that are identified by color, are unidentifiable from the view of a user with colorblindness. 

WCAG 2.0 Compliance: [1.4.1 Use of Color](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html): 

> Color is not used as the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element. (Level A)

Traceability: [Microassist Report 1 > WCAG Compliance > Links > Use of Color](https://docs.google.com/document/d/10cfUFx-o_kSGBFKCe4MtL45cyc-c-Gwaz3Dxlrp8Qgg/edit#bookmark=id.yo35si3ax4d4)

**Comment by paregorios** [comment]
We went to underline.
